api/desecapi/migrations/0020_user_email_verified.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.0.1 on 2022-01-14 13:39
+
+import datetime
+
+from django.db import migrations, models
+from django.db.models import F, Q
+
+
+def forwards_func(apps, schema_editor):
+    User = apps.get_model("desecapi", "User")
+    db_alias = schema_editor.connection.alias
+    filter = Q(is_active=True) | ~Q(last_login__isnull=True)
+    filter_kwargs = dict(created__date__gte=datetime.date(2019, 11, 1))
```

Comment:
What's the difference between `filter` and `filter_kwargs`? Couldn't they be one expression? Are their conditions ANDed or ORed?

Reply:
> Couldn't they be one expression?

No, because the Q objects (for doing OR) have to go to `filter()`'s `args`, and the date check has to go to `kwargs`.